frontend/src/components/AutoPrompt.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {FunctionComponent, ReactElement, useEffect} from "react";
+import PromptSpeech from "./PromptSpeech";
+
+interface SpeechProp {
+  prompt: string;
+}
+
+const AutoPrompt : FunctionComponent<SpeechProp> = ({ prompt }) : ReactElement => {
+
+  function talk() {
```

Comment:
small nitpick: i would do const talk = () => instead of talk()
1. for consistency 
2. This is an article that ive encountered before hehe https://medium.com/dailyjs/use-const-and-make-your-javascript-code-better-aac4f3786ca1